build-support/bin/native/bootstrap_code.sh

```diff
@@ -1,14 +1,20 @@
 REPO_ROOT="$(cd "$(dirname "${BASH_SOURCE[0]}")" && cd ../../.. && pwd -P)"
-
 # Defines:
 # + CACHE_ROOT: The pants cache directory, ie: ~/.cache/pants.
 # Exposes:
 # + die: Exit in a failure state and optionally log an error message to the console.
-# + fingerprint_data: Fingerprints the data on stdin.
-
 # shellcheck source=build-support/common.sh
 source "${REPO_ROOT}/build-support/common.sh"
 
+# Defines:
+# + NATIVE_ROOT: The Rust code directory, ie: src/rust/engine.
+# + MODE: Whether to run in debug or release mode.
+# + MODE_FLAG: The string to pass to Cargo to determine if we're in debug or release mode.
+# Exposes:
+# + calculate_current_hash: Generate a stable hash to determine if we need to rebuild the engine.
+# shellcheck source=build-support/bin/native/calculate_engine_hash.sh
```

Comment:
No. This is a hint to shellcheck for how to find the file because it does not understand `REPO_ROOT`. Shellcheck will use this to read in that file and determine if we're using any un-exposed symbols.

We use this in a couple other places.